src/components/molecules/DataResultItem.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Card, Pane, Heading } from 'evergreen-ui';
+import DatasetInfoBox from '../atoms/DatasetInfoBox';
+
+export type DataResultItemProps = {
+    title: string;
+    description: string;
+    tags: string[];
+};
+
+const DataResultItem: React.FC<DataResultItemProps> = ({ title, description, tags }) => {
+    const handleOnClick = () => {
+        return 0; //Will route to data customise page later.
+    };
+
+    return (
+        <Card width="100%" height="20rem" elevation={1} display="flex" onClick={handleOnClick}>
+            <Pane flex="1" padding="1rem">
+                <DatasetInfoBox title={title} description={description} tags={tags} />
+            </Pane>
+            <Pane flex="2">
+                <Heading>Forslag til visualisering</Heading>
+                {/* The template graph will go here */}
```

Comment:
Would it be possible to place the threshold chart here just as a placeholder? So that we know that the design works. Also as with the DashboardItem, shouldn't the graph be passed in as a child?

Reply:
I was thinking about just passing the charttype from metadata as a prop, since we will be faking the data anyways. Isn’t it more natural to handle all this in DataResultItem instead of the DataExplorer?

Reply:
I'd agree with Emil here, since you pass all required information to create a mocked chart from the metadata query, you can generate it here.

Reply:
It depends on where we think it is natural to "create" the chart. I suppose it makes more sense to "create" it in the Item, I just didn't see anywhere where the DataExplorer passed this information downwards.

Reply:
It's not passed in here, but it can be added when needed.

https://github.com/IT2901-SINTEF01/frontend/pull/62/files/fb0eb14a0e0d4e5e90c2c8c4b220290b37f36f46#diff-64daaaac0ccbbb9d38bd54d6d882e67c8784385bf68c39acc98085efb830b963R30-R34

Reply:
The information is already there, it's just not included in the proplist.

To make this easier @emilom, you could also pass the whole `el` as a prop to `DataResultItem` instead of destructuring it and extracting only a subset of fields.

Reply:
I.e.

```tsx
<DataResultItem metadata={el} />
```

Reply:
I didn’t add ThresholdChart as a placeholder now.  This can be added in a later issue when we implement the visualisation proposals.